statsmodels/tsa/stattools.py

```diff
@@ -1896,6 +1896,100 @@ def arma_order_select_ic(
     return Bunch(**res)
 
 
+def diebold_mariano_test(y, forecast1, forecast2, horizon=1,
```

Comment:
Need to have options for estimating the long-run variance of the loss differential.  When conducting DM tests it is necessary to allow for the possibility of a correlated sequence of loss differences.  Newey-west or something more general.